CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+
+* **Metrics**
+  * [UPGRADE] Kube-prometheus-stack has been upgraded from version 41.7.3 to 43.3.1
+  * [UPGRADE] Prometheus has been upgraded from version 2.39.0 to 2.40.7
+  * [UPGRADE] Prometheus Operator has been upgraded from version 0.60.0 to 0.62.0
+  * [UPGRADE] Grafana has been upgraded from version 9.2.3 to 9.3.1
+  * [UPGRADE] Alertmanager has been upgraded from version 0.24.0 to 0.25.0
+  * [UPGRADE] Node Exporter has been upgraded from version 1.3.1 to 1.5.0
+  * [UPGRADE] Kube State Metrics has been upgraded from version 2.6.0 to 2.7.0
+  * [UPGRADE] Pushgateway has been upgraded from version 1.4.3 to 1.5.1
+  * [UPGRADE] K8s-sidecar used with Grafana has been upgraded from 1.19.5 to 1.22.0
```

Comment:
For the cluster monitoring deployment, I do believe we're still just using whatever's associated with the kube-prometheus-stack.

For tenant and Openshift deployments, we do use a Grafana Helm chart value (which aligns with the version that we're updating to).  Interestingly enough, we've been identifying a separate Grafana Helm chart for the tenant and Openshift deployments since before I joined the team.